pkg/config/setup/high_availability.go

```diff
@@ -15,4 +16,14 @@ func setupHighAvailability(config pkgconfigmodel.Config) {
 	config.BindEnv("ha.dd_url")
 	config.BindEnvAndSetDefault("ha.enabled", false)
 	config.BindEnvAndSetDefault("ha.failover", false)
+
+	config.BindEnv("ha.remote_configuration.refresh_interval")
+	config.BindEnv("ha.remote_configuration.max_backoff_time")
```

Comment:
This config doesn't seem to be used anywhere ? You might be missing the `ha.` prefix in `rcserviceha.go` ?